projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-headline/order-detail-headline.component.html

```diff
@@ -24,4 +24,45 @@
       </div>
     </div>
   </div>
+  <div *ngIf="order?.orgCustomer" class="cx-header">
+    <div class="cx-detail">
+      <div class="cx-detail-label">
+        {{ 'orderDetails.purchaseOrderId' | cxTranslate }}
+      </div>
+      <div class="cx-detail-value">
+        {{
+          order?.purchaseOrderNumber
+            ? order?.purchaseOrderNumber
+            : ('orderDetails.emptyPurchaseOrderId' | cxTranslate)
+        }}
+      </div>
+    </div>
+    <div class="cx-detail">
+      <div class="cx-detail-label">
+        {{ 'orderDetails.placedBy' | cxTranslate }}
+      </div>
+      <div class="cx-detail-value">{{ order?.user?.name }}</div>
+    </div>
+    <div class="cx-detail">
+      <div class="cx-detail-label">
+        {{
+          order?.costCenter
+            ? ('orderDetails.costCenterAndUnit' | cxTranslate)
+            : ('orderDetails.costCenter' | cxTranslate)
+        }}
+      </div>
+      <div class="cx-detail-value">
+        {{
+          order?.costCenter
+            ? ('orderDetails.costCenterAndUnitValue'
+              | cxTranslate
+                : {
+                    costCenterName: order?.costCenter?.name,
+                    unitName: order?.orgCustomer?.orgUnit?.name
```

Comment:
nice catch, will verify if the b2bunit is the same in cost center and orgcustomer.orguser with Bill

Reply:
no need to fix this as Bill's new requirement says we do not display a unit for creditcard in the headline component.

It will be looking like this:
if account: display the 2 rows with 3 columns each
if creditcard: display the 2 rows with the top having the same 3 columns, and the bottom having 3 columns and exclude the bottom right (Unit), therefore a blank space.

Reply:
but I would need to use the costcenter.unit.name for the unit label/value

Reply:
NEW new requirement: instead of empty, display the same label and the value as none (paid by credit card)